docs/sources/flowspec.md

```diff
@@ -49,7 +49,7 @@ CLI syntax to add ipv4/ipv6 flowspec rule is
                     { port | destination-port | source-port | icmp-type | icmp-code | packet-length | dscp | label } <ITEM>... }...
    <PROTO> : ospf, pim, igp, udp, igmp, tcp, egp, rsvp, gre, ipip, unknown, icmp, sctp, <VALUE>
    <FRAGMENT_TYPE> : dont-fragment, is-fragment, first-fragment, last-fragment, not-a-fragment
-   <TCPFLAG> : rst, push, ack, urgent, fin, syn
+   <TCPFLAG> : U, C, E, F, S, R, P, A
```

Comment:
It is not very explicit but I will just mention = and ! in this section ... people can refer to flowspec.md examples to have a more detailed explanation of the syntax.